web/src/lib.rs

```diff
@@ -899,3 +910,18 @@ pub fn set_panic_handler() {
         }));
     });
 }
+
+fn populate_movie_parameters(
+    input: &JsValue,
+    output: &mut PropertyMap<String>,
+) -> Result<(), JsValue> {
+    let keys = js_sys::Reflect::own_keys(input)?;
+    for key in keys.values() {
+        let key = key?;
+        let value = js_sys::Reflect::get(input, &key)?;
+        if let (Some(key), Some(value)) = (key.as_string(), value.as_string()) {
+            output.insert(&key, value, false);
+        }
+    }
+    Ok(())
+}
```

Comment:
Would it be better if this just swallowed the errors instead of returning an Error, so that the movie still runs? (Not sure if this would ever really occur in practice).

Reply:
Addressed in 0bc6a314de6af81eb014c5d3a411d4e216c10eb9